components/python/salt/salt-PYVER.p5m

```diff
@@ -0,0 +1,1637 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2018 Till Wegmueller
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)-$(PYV)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value='Portable, distributed, remote execution and configuration management system'
+set name=info.classification value='org.opensolaris.category.2008:System/Administration and Configuration'
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='Apache License 2.0'
+
+# Tornado does a lot of try/import/except, making pkgdepend unhapyy
```

Comment:
1) Please, fix comment
2) What happens if you disable this hack?

Reply:
Done and done.